src/isar_turtlebot/turtlebot/taskhandlers/takethermalimage.py

```diff
@@ -82,7 +118,7 @@ def _move_status(self) -> Status:
         return move_status
 
     def _write_image_bytes(self):
-        image_data: str = self._get_image_data()
+        image_data: str = self.bridge.visual_inspection.get_image()
```

Comment:
The thermal image conversion should be added after it was removed from `robot_interface.py`. Consider adding it here, or await #43.

Reply:
Will be resolved by handling merge conflicts before merge.